feature/profile/src/main/kotlin/org/mifospay/feature/profile/ProfileScreen.kt

```diff
@@ -81,93 +66,46 @@ fun ProfileScreenContent(
 
             is ProfileUiState.Success -> {
                 ProfileImage(bitmap = profileState.bitmapImage)
-                Row(
+
+                ProfileDetailsCard(
+                    name = profileState.name ?: "Empty name",
```

Comment:
No need to set anything if name, email, mobile is nullable or empty, set only "" string like vpa if value is null or empty